packages/coordinator/src/components/Schedule/ScheduleWeek/ScheduleWeekComponent.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import styles from "./ScheduleWeekComponent.module.scss";
+import { ScheduleDay, ScheduleWeek } from "../../../utils/types";
+import _ from "lodash";
+import { DateUtils } from "@zm-blood-components/common";
+import ScheduleCellComponent from "../ScheduleCell";
+import classNames from "classnames";
+
+export type ScheduleWeekComponentProps = {
+  days: ScheduleDay[];
+};
+
+export default function ScheduleWeekComponent(
+  props: ScheduleWeekComponentProps
+) {
+  const days = props.days;
+  return (
+    <div className={styles.schedule}>
+      <div className={classNames(styles.gridColumns)}>
+        <div />
+        {days.map((day) => (
+          <DayHeader
+            day={day}
+            key={`day.${day.cells[0].cellStartTime.getTime()}`}
+          />
+        ))}
+      </div>
+
+      <div className={classNames(styles.scheduleWeek, styles.gridColumns)}>
+        {_.range(24).map((hourIndex) => {
```

Comment:
I think there is not need for all the 24 hours, maybe limit it to normal hours (like 8-20)..

Reply:
I agree, we should take this with Matar and Adi. Will keep it like it for now, and change later if we want to